src/testdrive/src/lib.rs

```diff
@@ -96,11 +116,13 @@ async fn run_line_reader(
     }
 
     if config.reset {
-        let mut errors = Vec::new();
+        // Clean up AWS state at the end of the run. Unlike Materialize and
```

Comment:
ty for this explanation; I had missed/misunderstood this context

Reply:
No no, it wasn't written anywhere. Just implicitly encoded across a bunch of testdrive scripts and compositions. 🙈